storage/persistent_storage/persistent-storage-cinder.adoc

```diff
@@ -16,11 +16,11 @@ requested by users.
 
 [IMPORTANT]
 ====
-{product-title} defaults to using an in-tree (non-CSI) plugin to provision Cinder storage.
+{product-title} 4.11 and later provides automatic migration for the Cinder in-tree volume plugin to its equivalent CSI driver.
 
-In future {product-title} versions, volumes provisioned using existing in-tree plugins are planned for migration to their equivalent CSI driver. CSI automatic migration should be seamless. Migration does not change how you use all existing API objects, such as persistent volumes, persistent volume claims, and storage classes. For more information about migration, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
+CSI automatic migration should be seamless. Migration does not change how you use all existing API objects, such as persistent volumes, persistent volume claims, and storage classes. For more information about migration, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
 
-After full migration, in-tree plugins will eventually be removed in future versions of {product-title}.
+In-tree plugins are expected to be removed in future versions of {product-title}.
```

Comment:
In-tree plugins are actually completely replaced by CSI migration. Whether they are removed or not in a future release should not matter, there is no way how to actually enable or use them.

Reply:
@gcharot We've been making this kind of statement for a while now. Were we making that statement generically about in-tree and migration. Should it be removed now?

Reply:
Yes agree it does not really matter and may actually raise questions.